packages/flutter/test/painting/text_painter_test.dart

```diff
@@ -12,11 +12,105 @@ const bool isCanvasKit =
     bool.fromEnvironment('FLUTTER_WEB_USE_SKIA');
 
 void main() {
+  void checkCaretOffsetsLtrAt(String text, List<int> boundaries) {
+    expect(boundaries.first, 0);
+    expect(boundaries.last, text.length);
+
+    final TextPainter painter = TextPainter()
+      ..textDirection = TextDirection.ltr;
+
+    // Lay out the string up to each boundary, and record the width.
+    final List<double> prefixWidths = <double>[];
+    for (final int boundary in boundaries) {
+      painter.text = TextSpan(text: text.substring(0, boundary));
+      painter.layout(maxWidth: 10000);
```

Comment:
nit: I think you could just omit `maxWidth` here? The default value is infinity iirc.

Reply:
Ah indeed, thanks for the catch.